### DISCUSSION TITLE
HomeAssistant & Frigate & Coral all in one on Intel NUC

### DISCUSSION THREAD

**Discussion by chokays** [blakeblackshear/frigate#4095] [Ask A Question]
My current setup is a Raspberry Pi 3 running HA and an Intel NUC with a Coral AI running Ubuntu with Frigate in a container. Works great but the Raspberry Pi is becoming the limiting factor for new home automation.s

I'm thinking of installing Home Assistant OS on the Intel nuc and running the Frigate docker on it as well.

Does anyone have experience with running this setup? For e.g. does the Coral AI card work under Home Assistant OS with Frigrate running in a docker?

**Comment by NickM-27** [comment]
Frigate will have to run as an addon in HA OS, the limitations of that setup are discussed in the docs

The coral should work without issue
